lab-doug/test/greet-test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
Excellent!  My only suggestion is to change line 3 to:
var greet = require(__dirname + '/../lib/greet').greet;
Just to make the function's origin more clear.  Can be helpful when debugging.